packages/ui-components/src/components/Icons.tsx

```diff
@@ -122,7 +122,8 @@ export enum UiIcons {
     Chevron = 'Chevron',
     GoToKey = 'GoToKey',
     File = 'File',
-    NoResults = 'NoResults'
+    NoResults = 'NoResults',
+    actionDetail = 'actionDetail'
```

Comment:
I see ```UiIcons``` is alphabetically sorted, and it follows the pascal case convention, maybe we can keep it like this.

Reply:
Thanks, its the wrong name entirely. Will rename.

Reply:
Still not sorted (most others are alphabetical as @unseen1980 mentioned).